client/transaction-pool/Cargo.toml

```diff
@@ -23,7 +23,7 @@ sc-transaction-graph = { version = "2.0.0-alpha.5", path = "./graph" }
 sp-transaction-pool = { version = "2.0.0-alpha.5", path = "../../primitives/transaction-pool" }
 sc-client-api = { version = "2.0.0-alpha.5", path = "../api" }
 sp-blockchain = { version = "2.0.0-alpha.5", path = "../../primitives/blockchain" }
-futures-timer = "2.0"
+intervalier = "0.3"
```

Comment:
Personally I'm not happy with having some extern crate that isn't reviewed. (I briefly checked the implementation).

AFAIU this crate provides primitives to send events when some interval finished. 

Wouldn't some sort of `validation_events` stream solve this better and may be resuable later for some other tasks as well? 

`validation_events` would always send an event when we re-validated the transaction pool?

Reply:
I don't think so, your approach will require a lot of confusing code that is not needed for production.

This stuff in this external crate is generic and reusable (I was missing this not only here). We don't audit everything (future-timer is not audited as well, for example).

Reply:
Yeah, I know that we don't audit everything :P I don't wanted to make your code look bad. If you say the other approach would be to complicated, fine.

Reply:
> Yeah, I know that we don't audit everything :P I don't wanted to make your code look bad. If you say the other approach would be to complicated, fine.

It is not that it is more complicated, it will pollute production code with something that is needed only for tests. This ^^^ almost does not pollute, except constructor.